examples/topics/osm-1billion.ipynb

```diff
@@ -40,7 +40,7 @@
    "outputs": [],
    "source": [
     "%%time\n",
-    "df = dd.io.parquet.read_parquet('../data/osm-1billion.snappy.parq')\n",
+    "df = SpatialPointsFrame.from_parquet('../data/osm-1billion.parq').persist()\n",
     "df = df.persist()"
```

Comment:
Persisting twice here.

Reply:
Oops; thanks!

Reply:
Fixed, and also hadn't uploaded the updated census.ipynb, also fixed.